### DISCUSSION TITLE
Plot view shouuld show job plot filename!

### DISCUSSION THREAD

**Discussion by bfasan** [swar/Swar-Chia-Plot-Manager#911] [Ideas]
It would be waay easier to fix/delete failed jobs and free up wasted space if  the plot job fille names were listed along with the pid to help identify files. currently i've been sleuthing which tmp files i think belong to failed jobs by file creation time and file name.. but it be way easier to just not have to guess..
maybe this could be in the next update?

**Comment by creamsoup** [comment]
you mean previous update? it is already there...

**Comment by bfasan** [comment]
Where does it show the plot filename? (Like ideally it would show with
manager.py view)

**Reply by creamsoup** [reply]
```
=======================================================================================================================================
num      job      k    plot_id     pid            start          elapsed_time   phase        phase_times        progress   temp_size
=======================================================================================================================================
16    chiaTemp1   32   53687d8   3460102   2021-05-27 13:58:14   00:06:59       1                               1.88%      47 GiB
```

see plot id.

**Reply by creamsoup** [reply]
okay it is not exactly filename, but you should be able to locate the file. 
`ls ${temp_dir}/*${plot_id}*`

**Comment by Ron-ski** [comment]
Its actually very easy to find the plot file, or temporary plot files, you simply match the start date/time with the files in the directory, using search can easily find all the files relating to that plot. As long as you don't have plots starting in the same minute (which you shouldn't) this will work.
